core/types/calls.cc

```diff
@@ -3010,6 +3010,31 @@ class Module_tripleEq : public IntrinsicMethod {
     }
 } Module_tripleEq;
 
+class T_Enum_tripleEq : public IntrinsicMethod {
```

Comment:
Note that the implementation is based on that of `Module_tripleEq` above.

Reply:
Oh, `Module_tripleEq` wasn't firing because the receiver was a value of one of the generated enum classes?

Reply:
Yup!